api/controllers/user.js

```diff
@@ -1,7 +1,12 @@
+const auth = require('../middleware/authorization');
 const bcrypt = require('bcrypt');
 const mongoose = require('mongoose');
 const { User, validateUser } = require('../models/user');
 
+exports.usersGetMe = async (req, res, next) => {
+    const user = await User.findById(req.user._id).select('-password');
+    res.send(user);
+}
```

Comment:
Wszystko wygląda ok (przetestowałam wszystko również na userCompanyEndpoints).
Jedyne co to w controllers/user dla usersGetMe dodałabym sprawdzenie czy user istnieje (możesz zobaczyć jak u mnie to wygląda).

Reply:
Słuszna uwaga, już dodałem :)